neptune/management/internal/api.py

```diff
@@ -433,3 +437,140 @@ def get_workspace_member_list(name: str, api_token: Optional[str] = None) -> Dic
         }
     except HTTPNotFound as e:
         raise WorkspaceNotFound(workspace=name) from e
+
+
+@with_api_exceptions_handler
+def _get_raw_workspace_service_account_list(
+    name: str, api_token: Optional[str] = None
+) -> Dict[str, ServiceAccountDTO]:
+    verify_type("name", name, str)
+    verify_type("api_token", api_token, (str, type(None)))
+
+    backend_client = _get_backend_client(api_token=api_token)
+
+    params = {"organizationIdentifier": name, "deactivated": False, **DEFAULT_REQUEST_KWARGS}
+
+    try:
+        result = backend_client.api.listServiceAccounts(**params).response().result
+        return {f"{sa.name}": ServiceAccountDTO(name=sa.name, id=sa.id) for sa in result}
+    except HTTPNotFound as e:
+        raise WorkspaceNotFound(workspace=name) from e
+
+
+@with_api_exceptions_handler
+def get_workspace_service_account_list(
+    name: str, api_token: Optional[str] = None
+) -> Dict[str, str]:
+    service_accounts = _get_raw_workspace_service_account_list(name=name, api_token=api_token)
+
+    return {
+        service_account_name: WorkspaceMemberRoleDTO.to_domain("member")
+        for service_account_name, _ in service_accounts.items()
+    }
+
+
+@with_api_exceptions_handler
+def get_project_service_account_list(
+    name: str, workspace: Optional[str] = None, api_token: Optional[str] = None
+) -> Dict[str, str]:
+    verify_type("name", name, str)
+    verify_type("workspace", workspace, (str, type(None)))
+    verify_type("api_token", api_token, (str, type(None)))
+
+    backend_client = _get_backend_client(api_token=api_token)
+    project_identifier = normalize_project_name(name=name, workspace=workspace)
+
+    params = {"projectIdentifier": project_identifier, **DEFAULT_REQUEST_KWARGS}
+
+    try:
+        result = backend_client.api.listProjectServiceAccounts(**params).response().result
+        return {
```

Comment:
You return dict from `get_project_service_account_list` function. Change one of that :p

Reply:
Huh? This is `get_project_service_account_list`